lib/npm/install.js

```diff
@@ -82,6 +83,17 @@ function install(context, next) {
       context.testError = installError || '';
       return next(Error('Install Failed'));
     }
+    if (context.module.verifyNodeGyp !== undefined) {
+      var nodeGypCalled = fs.existsSync(path.join(context.path,
```

Comment:
would really like this to not be done with Sync, in fact I feel like there shoudl be a way to do this through printing to stdout rather than relying on a file to pass data